Updates/0035_summoning_the_princess.sql

```diff
@@ -0,0 +1,66 @@
+DELETE FROM dbscripts_on_quest_start WHERE id = 692;
+INSERT INTO `dbscripts_on_quest_start` (`id`, `delay`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `dataint`, `x`, `y`, `z`, `o`, `comments`) VALUES 
+('692', '0', '3', '0', '0', '0', '0', '0', '0', '0', '0', '4.2935', 'The Lost Fragments - Theldurin - Set Orientation'),
+('692', '0', '1', '25', '0', '0', '0', '0', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Emote OneShotPoint'),
+('692', '0', '0', '0', '0', '0', '0', '3004', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Say Text'),
+('692', '4000', '3', '0', '0', '0', '0', '0', '0', '0', '0', '1.04719', 'The Lost Fragments - Theldurin - Set Orientation');
+
+DELETE FROM dbscripts_on_quest_end WHERE id = 692;
+INSERT INTO `dbscripts_on_quest_end` (`id`, `delay`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `dataint`, `x`, `y`, `z`, `o`, `comments`) VALUES 
+('692', '0', '0', '0', '0', '0', '0', '870', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Emote Text'),
+('692', '4000', '3', '0', '0', '0', '0', '0', '0', '0', '0', '4.18879', 'The Lost Fragments - Theldurin - Set Orientation'),
+('692', '4000', '28', '8', '0', '0', '0', '0', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - StandState Kneel'),
+('692', '6000', '28', '0', '0', '0', '0', '0', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - StandState Stand'),
+('692', '7000', '0', '0', '0', '0', '0', '871', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Say Text'),
+('692', '10000', '3', '0', '0', '0', '0', '0', '0', '0', '0', '6.1686558', 'The Lost Fragments - Theldurin - Set Orientation'),
+('692', '10000', '42', '0', '0', '0', '0', '1911', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Weapon'),
+('692', '13000', '1', '4', '0', '0', '0', '0', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Emote OneShotCheer'),
+('692', '13000', '0', '0', '0', '0', '0', '872', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Say Text'),
+('692', '16000', '42', '1', '0', '0', '0', '0', '0', '0', '0', '0', 'The Lost Fragments - Theldurin - Default Weapon');
+
+UPDATE `quest_template` SET `StartScript`=692, `CompleteScript`=692 WHERE `entry`=692;
+UPDATE `quest_template` SET `DetailsEmote1`='0', `DetailsEmote2`='0', `DetailsEmote3`='0', `DetailsEmote4`='0' WHERE (`entry`='692');
+UPDATE `broadcast_text` SET `ChatTypeID`='2' WHERE (`Id`='870');
+
+-- Summoning the Princess
+UPDATE `quest_template` SET `DetailsEmote1`='0', `DetailsEmote2`='0', `DetailsEmote3`='0', `DetailsEmote4`='0' WHERE (`entry`='656');
+
+DELETE FROM dbscripts_on_event WHERE id = 420;
+INSERT INTO `dbscripts_on_event` (`id`, `priority`, `delay`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `data_flags`, `dataint`, `x`, `y`, `z`, `o`, `comments`) VALUES 
+('420', '0', '0', '10', '2755', '3000000', '0', '0', '0','0', '-931.73', '-3111.81', '48.517', '3.27404', 'Summoning the Princess - Summon Myzrael'),
```

Comment:
The second questions is about this line.
After using the item  [Scroll of Myzrael]  id 4472, the npc gets spawned. I watched multiple videos and did this quest on retail tbc and it seems this dont get spawned on fixed point, instead it mostly got spawned infront of the Player.
Will need more research for this.

Reply:
This generally indicates use of a spell with unit field created_by_spell

Reply:
The Spell (4141) use Id: 61 (SPELL_EFFECT_SEND_EVENT) as Effect0. 
I watched some more videos and i think it was just luck that it spawned directly next to me. 

In this 2 videos she has the same coords as in my personal sniff from retail tbc. 
https://www.youtube.com/watch?v=g9QlmT3hzlw
https://www.youtube.com/watch?v=g9QlmT3hzlw

Reply:
Then give it the same exact coords and dont worry about it. Dont need to overcomplicate things for ourselves.

Reply:
Yeah im sorry, just always want to have everything perfect :x